ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -1884,3 +1884,174 @@ def _distributed(self) -> bool:
+        """Wrap the ``csyiqr`` method to take advantage of the gRPC methods."""
+        pname = '__tmpvar__'
+        super().csyiqr(ncsy, key, pname=pname, mute=True, **kwargs)
+        value = self.scalar_param(pname)
+        self._run(f'{pname}=') # Deleting temp parameter
+        return value
+
+    @wraps(_MapdlCore.etyiqr)
+    def etyiqr(self, itype, key, **kwargs):
+        """Wrap the ``etyiqr`` method to take advantage of the gRPC methods."""
+        pname = '__tmpvar__'
+        super().etyiqr(itype, key, pname=pname, mute=True, **kwargs)
+        value = self.scalar_param(pname)
+        self._run(f'{pname}=') # Deleting temp parameter
+        return value
+
+    @wraps(_MapdlCore.foriqr)
+    def foriqr(self, node, key, **kwargs):
+        """Wrap the ``foriqr`` method to take advantage of the gRPC methods."""
+        pname = '__tmpvar__'
+        super().foriqr(node, key, pname=pname, mute=True, **kwargs)
+        value = self.scalar_param(pname)
+        self._run(f'{pname}=') # Deleting temp parameter
+        return value
+
+    @wraps(_MapdlCore.sectinqr)
+    def sectinqr(self, nsect, key, **kwargs):
+        """Wrap the ``sectinqr`` method to take advantage of the gRPC methods."""
+        pname = '__tmpvar__'
+        super().sectinqr(nsect, key, pname=pname, mute=True, **kwargs)
+        value = self.scalar_param(pname)
+        self._run(f'{pname}=') # Deleting temp parameter
+        return value
+
+    @wraps(_MapdlCore.mpinqr)
+    def mpinqr(self, mat,  iprop,  key, **kwargs):
+        """Wrap the ``mpinqr`` method to take advantage of the gRPC methods."""
+        pname = '__tmpvar__'
+        super().mpinqr(mat,  iprop,  key, pname=pname, mute=True, **kwargs)
+        value = self.scalar_param(pname)
+        self._run(f'{pname}=') # Deleting temp parameter
+        return value
+
+    @wraps(_MapdlCore.dget)
+    def dget(self, node,  idf,  kcmplx, **kwargs):
+        """Wrap the ``dget`` method to take advantage of the gRPC methods."""
+        pname = '__tmpvar__'
+        super().dget(node,  idf,  kcmplx, pname=pname, mute=True, **kwargs)
+        value = self.scalar_param(pname)
+        self._run(f'{pname}=') # Deleting temp parameter
+        return value
+
+    @wraps(_MapdlCore.fget)
+    def fget(self, node,  idf,  kcmplx, **kwargs):
+        """Wrap the ``fget`` method to take advantage of the gRPC methods."""
+        pname = '__tmpvar__'
+        super().fget(node,  idf,  kcmplx, pname=pname, mute=True, **kwargs)
+        value = self.scalar_param(pname)
+        self._run(f'{pname}=') # Deleting temp parameter
+        return value
+
+    @wraps(_MapdlCore.erinqr)
+    def erinqr(self, key, **kwargs):
+        """Wrap the ``erinqr`` method to take advantage of the gRPC methods."""
+        pname = '__tmpvar__'
+        super().erinqr(key, pname=pname, mute=True, **kwargs)
+        value = self.scalar_param(pname)
+        self._run(f'{pname}=') # Deleting temp parameter
```

Comment:
Delete this for all methods please. There's no reason to delete a temp parameter.

Reply:
Ok. Done.